### PR TITLE
adding warning to Making content editable

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/mdn/content/issues/1444

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
@Ryuno-Ki warning added. What do you think?

@schalkneethling have I used the correct markup for the warning box?

**Comment by schalkneethling** [comment]
> @schalkneethling have I used the correct markup for the warning box?

100% - The styling for heading levels h2, h3, and h4 will land tomorrow in Yari.

**Review by schalkneethling — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
@schalkneethling yay, cheers!

File: files/en-us/web/guide/html/editable_content/index.html [hunk 1]
```diff
@@ -15,6 +15,11 @@
   - text entry
   - text input
 ---
+<div class="notecard warning">
+  <h2>Warning</h2>
+  <p>As the <a href="https://w3c.github.io/editing/docs/execCommand/"><code>execCommand()</code> spec</a> warns, its features <q>are not implemented consistently or fully by user agents</q>, and in addition, it is marked as deprecated on the <a href="/en-US/docs/Web/API/Document/execCommand"><code>Document.execCommand()</code></a> reference page. Therefore, much of the content on this page cannot be trusted for use in production code.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/guide/html/editable_content/index.html:20]: Could you add an `external` class?
According to https://github.com/mdn/mdn-minimalist/issues/458#issuecomment-768134750 that would be the way to flag external links going forwards.

**Comment by Ryuno-Ki** [comment]
> @Ryuno-Ki warning added. What do you think?

Added a nitpick.

On a side note: Would [`document.designMode`](https://developer.mozilla.org/en-US/docs/Web/API/Document/designMode) fall into the same category?

File: files/en-us/web/guide/html/editable_content/index.html (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/guide/html/editable_content/index.html:20]: I believe the system will add these automatically on build, to any external links.

**Pull request merged**